parsing_test.go

```diff
@@ -28,16 +28,19 @@ Some more content
 `,
 		"",
```

Comment:
Maybe add a test with an assumption to verify it is parsed and the prefix is correct